packages/myst-to-typst/src/index.ts

```diff
@@ -381,12 +394,6 @@ class TypstSerializer implements ITypstSerializer {
     this.trimEnd();
     this.file.result += ']';
   }
-
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  closeBlock(node: any) {
-    this.ensureNewLine(true);
-    this.file.result += '\n';
-  }
```

Comment:
I think that we are loosing the block information between paragraphs with this change @fwkoch.